docker_files/recipes/openfoam.pimplefoam-2.4.0/build.sh

```diff
@@ -16,9 +16,15 @@ echo "/usr/bin/pimpleFoam.so: ${OSV_DIR}/mike-apps/OpenFOAM/ROOTFS/usr/bin/pimpl
 
 # Patch the module file to exclude the dependency on HTTP server
 cd ${OSV_DIR}
+patch -p1 < ${COMMON_DIR}/openfoam_core.patch
 patch -p1 < ${COMMON_DIR}/openfoam_remove_ompi.patch
 ${OSV_DIR}/scripts/build image=OpenFOAM export=all usrskel=none export_dir=$PACKAGE_RESULT_DIR -j ${CPU_COUNT}
 
+# Add simpleFoam solver
+cp ${OSV_DIR}/mike-apps/OpenFOAM/ROOTFS/usr/bin/pimpleFoam.so ${PACKAGE_RESULT_DIR}/usr/bin
+rm ${PACKAGE_RESULT_DIR}/usr/bin/decomposePar
```

Comment:
Looking at the [core script](https://github.com/gasper-vrhovsek/capstan-packages/blob/c91a8ea373bfbd547b5e86e2497c7f11c58121b7/docker_files/recipes/openfoam.core-2.4.0/build.sh#L22-L23)  `decomposePar` and `reconstructPar` are not actually copied to `/usr/bin` be default.

Are there `rm`s mandatory?

Reply:
It weren't the `decomposePar.so` and `reconstructPar.so` that were the problems, but the symbolic links that get created and should point to those two files, but in this case pointed to nowhere.

Reply:
In the result logs these lines get printed out:
```
  940: [INFO] added link /result/intermediate/openfoam.pimplefoam-2.4.0/usr/bin/decomposePar -> decomposePar.so
  941  [INFO] added link /result/intermediate/openfoam.pimplefoam-2.4.0/usr/bin/reconstructPar -> reconstructPar.so
```
I'll try to find where this happens. If possible i'll implement a better solution.

Reply:
Thanks to @miha-plesko we found in which bush the rabbit is hiding:
https://github.com/mikelangelo-project/mike-apps/blob/master/OpenFOAM/GET#L132-L134

:)

Reply:
As discovered offline with @gasper-vrhovsek the problem is that [GET script](https://github.com/mikelangelo-project/mike-apps/blob/master/OpenFOAM/GET#L132-L134) of the mike-apps/OpenFOAM creates these two links. We agreed that it's best to fix the GET script and remove the `rm`s from here.